modules/core/src/test/scala/fs2/kafka/KafkaConsumerSpec.scala

```diff
@@ -516,6 +516,7 @@ final class KafkaConsumerSpec extends BaseKafkaSpec {
               }
             }
             .takeWhile(_.size < 200)
+            .timeout(20.seconds)
```

Comment:
Timeout added to match test with custom assignor. This test was working without it, while the one for the custom assignor would hang indefinitely.